packages/salesforcedx-vscode-core/src/conflict/metadataCacheService.ts

```diff
@@ -310,12 +316,18 @@ export class MetadataCacheService {
    * @param index The map which is mutated by this function
    * @param components The parent and/or child components to add to the map
    */
-  private static pairParentsAndChildren(index: Map<string, RecomposedComponent>, components: SourceComponent[]) {
+  private static pairParentsAndChildren(
+    index: Map<string, RecomposedComponent>,
+    components: SourceComponent[]
+  ) {
     for (const comp of components) {
       const key = MetadataCacheService.makeKey(comp.type.name, comp.fullName);
       // If the component has a parent it is assumed to be a child
       if (comp.parent) {
-        const parentKey = MetadataCacheService.makeKey(comp.parent.type.name, comp.parent.fullName);
+        const parentKey = MetadataCacheService.makeKey(
+          comp.parent.type.name,
+          comp.parent.fullName
+        );
```

Comment:
just to verify have you run `npm run prettier` at the top level? Seems odd all these format changes are showing up.

Reply:
I just ran it and got no output or changes.  Seems weird.  Yes, I am seeing a significant number of formatting only changes that only show up when I save a file locally.